### PR TITLE
Preparation job

### PR DESCRIPTION
**Pull request by kristianschneider**
This PR separates the ingestion of data out of the aggregation and wholesale job.
Input data is now stored as a snapshot and used in the following jobs.

### CODE REVIEW

File: source/databricks/aggregation-jobs/preparation_trigger.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+from geh_stream.aggregation_utils.aggregators import \
+    load_metering_points, \
+    load_time_series, \
+    load_market_roles, \
+    load_es_brp_relations, \
+    load_charges, \
+    load_charge_links, \
+    load_charge_prices, \
+    load_grid_loss_sys_corr, \
+    initialize_spark
```
Comment (ZavezX) [inline source/databricks/aggregation-jobs/preparation_trigger.py:33]: We should use the shared `data_loader` instead.

**Review by ZavezX — CHANGES_REQUESTED** [review]

**Comment by johevemi** [comment]
need snapshotId

File: source/databricks/geh_stream/shared/services/coordinator_service.py [hunk 2]
```diff
@@ -20,8 +20,6 @@
 class CoordinatorService:
 
     def __init__(self, args):
-        self.coordinator_url = args.result_url
-        self.snapshot_url = args.snapshot_url
```
Comment (johevemi) [inline source/databricks/geh_stream/shared/services/coordinator_service.py:24]: Needs a snapshotId

File: source/databricks/geh_stream/shared/services/coordinator_service.py [hunk 3]
```diff
@@ -20,8 +20,6 @@
...
         self.result_id = args.result_id
```
Comment (johevemi) [inline source/databricks/geh_stream/shared/services/coordinator_service.py:23]: We also need a snapshotId in the header https://github.com/Energinet-DataHub/geh-aggregations/blob/2f7de5d85daf989dcc9a393b759ef8266383442e/source/databricks/geh_stream/shared/services/coordinator_service.py#L33 for the snapshotReciver in the dotnet code for us to update the path on a given snapshot

Comment (kristianschneider) [reply source/databricks/geh_stream/shared/services/coordinator_service.py:23]: done

**Review by johevemi — APPROVED** [review]

**Review by ZavezX — CHANGES_REQUESTED** [review]
Arguments sent from the coordinator functions have changed. 
[trigger_base_arguments](https://github.com/Energinet-DataHub/geh-aggregations/blob/c96dccbb4211c8d0cacec6ec6e24922105bd381c/source/databricks/geh_stream/aggregation_utils/trigger_base_arguments.py) needs to be updated to correspond to the arguments sent from [here](https://github.com/Energinet-DataHub/geh-aggregations/blob/c96dccbb4211c8d0cacec6ec6e24922105bd381c/source/coordinator/GreenEnergyHub.Aggregation.Coordinator/CoordinationTriggers.cs#L319), [here](https://github.com/Energinet-DataHub/geh-aggregations/blob/c96dccbb4211c8d0cacec6ec6e24922105bd381c/source/coordinator/GreenEnergyHub.Aggregation.Coordinator/CoordinationTriggers.cs#L356) and [here](https://github.com/Energinet-DataHub/geh-aggregations/blob/c96dccbb4211c8d0cacec6ec6e24922105bd381c/source/coordinator/GreenEnergyHub.Aggregation.Coordinator/CoordinationTriggers.cs#L294)

**Review by ZavezX — APPROVED** [review]

**Pull request merged**